jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/model/Options.java

```diff
@@ -0,0 +1,70 @@
+package de.jplag.end_to_end_testing.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class manages the test options and thus allows testing of different JPlag options for the test cases.
+ */
+public class Options {
```

Comment:
This can be a record

Reply:
Jackson JSON should be able to use records.

Reply:
class was replaced by record